interface/forms/soap/templates/general_new.html

```diff
@@ -1,52 +1,58 @@
-                            <textarea name="plan" id="plan" class="form-control" cols="60" rows="6">{$data->get_plan()|escape:'html'}</textarea>
-                        </div>
-                    </div>
-                    <div class="form-group">
-                        <div class="col-sm-offset-2 col-sm-10">
-                            <div class="btn-group" role="group">
+                <form name="soap" method="post" action="{$FORM_ACTION}/interface/forms/soap/save.php" onsubmit="return top.restoreSession()">
+                    <fieldset>
+                            <legend class="">{xl t='Subjective'|escape:'html'}</legend>
+                                <div class="form-group" >
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="subjective" class="form-control" cols="40" rows="6">{$data->get_subjective()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <fieldset>
+                            <legend class="">{xl t='Objective'|escape:'html'}</legend>
+                                <div class="form-group">
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="objective" class="form-control" cols="60" rows="6">{$data->get_objective()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <fieldset>
+                            <legend class="">{xl t='Assessment'|escape:'html'}</legend>
+                                <div class="form-group">
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="assessment" class="form-control" cols="60" rows="6">{$data->get_assessment()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <fieldset>
+                            <legend class="">{xl t='Plan'|escape:'html'}</legend>
+                                <div class="form-group">
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="plan" class="form-control" cols="60" rows="6">{$data->get_plan()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <?php //can change position of buttons by creating a class 'position-override' and adding rule text-align:center or right as the case may be in individual stylesheets ?>
+                    <div class="form-group clearfix">
+                        <div class="col-sm-12 text-left position-override">
```

Comment:
This tests really well in the SOAP form, which I am focusing on. By pulling this to the left rather than lining it up with the input elements, these buttons are actually ending up in the worst place (according to those button articles).